constants/map_object_constants.asm

```diff
@@ -31,13 +31,13 @@ DEF OBJECT_MOVEMENT_BYTE_INDEX rb ; 1b
 DEF OBJECT_1C                  rb ; 1c
 DEF OBJECT_1D                  rb ; 1d
 DEF OBJECT_1E                  rb ; 1e
```

Comment:
These seem to be used for a couple purposes... One of which seems to store the address of the start of another object used in `StepFunction_TrackingObject`. So it's fine they remain as they are now.. just wanted to mention it.